en, zh: add clarify for additionalVolumeMounts (#1187)
pingcap/docs-tidb-operator: en/backup-to-pv-using-br.md

PVs in this documentation can be any [Kubernetes supported Persistent Volume types](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes). This document uses NFS as an example PV type.

> > **Warning:**

Reviewer: ```suggestion
> **Note:**
```

>

> > The backup method described in this document is supported since TiDB Operator v1.1.8 version.

Reviewer: ```suggestion
> The backup method described in this document is supported starting from TiDB Operator v1.1.8.
```
